definitions/npm/react-splitter-layout_v3.x.x/test_splitter-layout.js

```diff
@@ -0,0 +1,93 @@
+import { describe, it } from 'flow-typed-test';
+import React from 'react';
+import SplitterLayout from 'react-splitter-layout';
+
+it('checks it takes at least 1 pane', () => {
+  let element = (
+    <SplitterLayout>
+      <div>pane 1</div>
+      <div>pane 2</div>
+    </SplitterLayout>
+  );
+
+  element = (
+    <SplitterLayout>
+      <div>pane 1</div>
+    </SplitterLayout>
+  );
+
+  element = (
+    <SplitterLayout>
+      <div>pane 1</div>
+      {undefined}
+    </SplitterLayout>
+  )
+
+  element = (
+    <SplitterLayout>
+      {null}
+      <div>pane 1</div>
```

Comment:
So, in previous versions of this definition, I used something like `React$Element | [ React$Element | null, React$Element | null ]` (not exactly this but similar) and so I wanted to make sure that `null` was tested. I can remove them but I feel like this could be a good defensive test for future changes.

Please tell me what you think, I'd be happy to remove these tests if you still think they're useless :)